src/test/java/co/org/osso/api/ApliApplicationCalculatorTests.java

```diff
@@ -0,0 +1,60 @@
+package co.org.osso.api;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ApliApplicationCalculatorTests {
+
+    CalculatorService CalculatorServiceTarget = new CalculatorService();
+
+    @Test
+    public void checkCalculatorServiceFibonacci(){
+        List<Integer> fib = new ArrayList<>();
+        fib.add(1);
+        fib.add(1);
+        fib.add(2);
+        fib.add(3);
+        fib.add(5);
+        Assert.assertEquals(fib, CalculatorServiceTarget.getFibonacci(5));
+    }
+
+    @Test
+    public void checkCalculatorServiceFactorial(){
+        Assert.assertEquals(3628800, CalculatorServiceTarget.getFactorial(10));
+    }
+
+    @Test
+    public void checkCalculatorServiceOperationsSum(){
+        Assert.assertEquals((double)12.2,CalculatorServiceTarget.getOperationResult("sum", 9.1, 3.1),.0);
```

Comment:
You should have a larger delta than `0.0` as the accuracy of this might vary per system.
Java isn't entirely portable here unless you use a keyword called `strictfp`. Congratulations, you're now among the very few Java developers who knows about this keyword. You'll probably never see it again lol.

`0.001` is probably fine in this case.